.github/workflows/playwright-e2e.yml

```
@@ -0,0 +1,139 @@
+    with:
+      pr: ${{ inputs.pr }}
+      branch: ${{ inputs.branch }}
+
+  server-build:
+    needs: path-filter
+    if: success() && needs.path-filter.outputs.not-ci == 'true' && inputs.docker_image_name == ''
+    uses: ./.github/workflows/server-build.yml
+    secrets: inherit
+    with:
+      pr: ${{ inputs.pr }}
+      skip-tests: true
```

actionlint (1.7.12)

[error] input "skip-tests" is typed as string by reusable workflow "./.github/workflows/server-build.yml". bool value cannot be assigned

(expression)